README.md

```diff
@@ -82,17 +82,17 @@ brew install azure-functions-core-tools
 ### Linux
 
 1. Set up package feed
-    * Ubuntu 18.04
+    * Ubuntu 18.10
 
         ```bash
-        wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
+        wget -q https://packages.microsoft.com/config/ubuntu/18.10/packages-microsoft-prod.deb
         sudo dpkg -i packages-microsoft-prod.deb
         ```
 
-    * Ubuntu 17.10
+    * Ubuntu 18.04
```

Comment:
Is 17.10 no longer supported?

Reply:
It's not worth listing all possible versions of linux. There's plenty not listed here that are still supported. Ultimately I just copy this list from here: https://github.com/Azure/azure-functions-core-tools/blob/dev/README.md